components/ColorPicker/src/CommunityToolkit.WinUI.Controls.ColorPicker.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="MSBuild.Sdk.Extras/3.0.23">
+  <PropertyGroup>
+    <ToolkitComponentName>ColorPicker</ToolkitComponentName>
+    <Description>This package contains ColorPicker.</Description>
+
+    <!-- Rns suffix is required for namespaces shared across projects. See https://github.com/CommunityToolkit/Labs-Windows/issues/152 -->
+    <RootNamespace>CommunityToolkit.WinUI.Controls.ColorPickerRns</RootNamespace>
+  </PropertyGroup>
```

Comment:
@niels9001 there's the package name override missing here like the other Controls packages, so the package is being published as `CommunityToolkit.Uwp.ColorPicker` instead of `CommunityToolkit.Uwp.Controls.ColorPicker` for instance.